tests/testthat/test-are-all-close.R

```diff
@@ -0,0 +1,51 @@
+test_that("are_all_close_correct_true", {
```

Comment:
try to have the filename match the function, so are_all_close and not are-all-close